ieproxy_windows.go

```diff
@@ -76,7 +76,25 @@ func parseRegedit(regedit regeditValues) ProxyConf {
 }
 
 func readRegedit() (values regeditValues, err error) {
-	k, err := registry.OpenKey(registry.CURRENT_USER, `Software\Microsoft\Windows\CurrentVersion\Internet Settings`, registry.QUERY_VALUE)
+	// Check config per user or per machine
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `Software\Policies\Microsoft\Windows\CurrentVersion\Internet Settings`, registry.QUERY_VALUE)
+	if err != nil {
+		return
+	}
+	defer k.Close()
```

Comment:
This will cause `Close()` to be called twice on the first value of `k` if line 95 is passed. E.g.: https://play.golang.org/p/uoG0URj-wNg